modules/README.md

```diff
@@ -101,3 +101,33 @@ $ cmake -D OPENCV_EXTRA_MODULES_PATH=<opencv_contrib>/modules -D BUILD_opencv_<r
 - **xobjdetect**: Boosted 2D Object Detection -- Uses a Waldboost cascade and local binary patterns computed as integral features for 2D object detection.
 
 - **xphoto**: Extra Computational Photography -- Additional photo processing algorithms: Color balance / Denoising / Inpainting.
+
+- **cudaarithm**: GPU-accelerated operations for image processing and computer vision -- Arithmetic operations / Bitwise operations / Logical operations.
```

Comment:
Please keep lexicographical order

Reply:
sorry i am new so idk what is lexicographical order is it same as ABCD or the order was given in the issue https://github.com/opencv/opencv_contrib/issues/2600